model/metricContributors/percentile/pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>pentaho</groupId>
     <artifactId>pentaho-data-profiling-model</artifactId>
     <version>6.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
+    <relativePath>../../pom.xml</relativePath>
```

Comment:
No, this modification was done because the relativePath was wrong. Before because we did not set the dependencyManagement, pluginManagement and properties in the project parent pom, it did not caused issues.